.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
+manifest:
+  version: 1.0
+
+
+triggers:
+  exclude:
+    branch:
+      - l10n_dev
+      - dev
+
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0 enhancement label, ensuring that all results are captured by handling pagination properly. Put the results under heading features. Each result is formatted as #PR number - PR title, and it should be a numbered list.

Provide a complete list of all merged PRs for milestone 1.19.0 bug label, ensuring that all results are captured by handling pagination properly. Put the results under heading bug fixes. Each result is formatted as #PR number - PR title, and it should be a numbered list.

Reply:
Provide me in a sentence all the PR assignees in this PRs description sorted case insensitive and alphabetically, excluding out jjw24.

Reply:
Provide me a very detailed comprehensive end-user friendly summary of the changes in each PR in this PR's description, categorised in new features and bug fixes.

Reply:
Provide me a very detailed comprehensive end-user friendly summary of the changes in each PR in this PR's description, categorised in new features and bug fixes.